webui/react/src/pages/ExperimentList.tsx

```diff
@@ -80,13 +81,123 @@ const ExperimentList: React.FC = () => {
     },
   );
   const initSorter = storage.getWithDefault(STORAGE_SORTER_KEY, { ...defaultSorter });
-  const [ pagination, setPagination ] = useState<Pagination>({ limit: initLimit, offset: 0 });
-  const [ total, setTotal ] = useState(0);
+  const [ experiments, setExperiments ] = useState<ExperimentItem[]>();
   const [ filters, setFilters ] = useState<ExperimentFilters>(initFilters);
-  const [ sorter, setSorter ] = useState(initSorter);
+  const [ isUrlParsed, setIsUrlParsed ] = useState<boolean>(false);
+  const [ pagination, setPagination ] = useState<Pagination>({ limit: initLimit, offset: 0 });
   const [ search, setSearch ] = useState('');
-  const [ experiments, setExperiments ] = useState<ExperimentItem[]>();
   const [ selectedRowKeys, setSelectedRowKeys ] = useState<string[]>([]);
+  const [ sorter, setSorter ] = useState(initSorter);
+  const [ total, setTotal ] = useState(0);
+
+  /*
+   * When filters changes update the page URL.
+   */
+  useEffect(() => {
+    if (!isUrlParsed) return;
+
+    const searchParams: Record<string, string> = {
+      archived: filters.showArchived ? '1' : '0',
+      limit: pagination.limit.toString(),
+      offset: pagination.offset.toString(),
+      sortDesc: sorter.descend ? '1' : '0',
```

Comment:
should this only be set if `sortKey` is set?

Reply:
We discussed via IM that all params has to be set in the URL bc otherwise an unset param will cause the page to be rendered with the user preference (that might be different from the global default value).